src/main/java/uk/gov/hmcts/reform/cwrdapi/domain/UserType.java

```diff
@@ -45,4 +45,9 @@
     @OneToMany(targetEntity = CaseWorkerProfile.class, mappedBy = "userType")
     private List<CaseWorkerProfile> caseWorkerProfiles = new ArrayList<>();
 
+    public UserType(Long userTypeId, String description) {
```

Comment:
Can't we use the builder pattern to create objects instead of adding a constructor?